src/main/webapp/script.js

```diff
@@ -483,9 +483,13 @@ function createPageElement(forumPage, pageNumber, searchString) {
   if (forumPage.previousPage) {
     previousWrapper.setAttribute('class', 'page-item');
     if (searchString != '') {
-      nextWrapper.onclick = fetchForum(pageNumber - 1);
+      previousWrapper.onclick = function() {
+        fetchForum(pageNumber - 1);
```

Comment:
Does this work on the first page? What happens if they click the previous button and go back to the 0 or -1 page?

Reply:
Oh just kidding I just saw the earlier code from a different review. Nevermind!